config/locales/models/en.yml

```diff
@@ -11,6 +11,16 @@ en:
             user_defined_tags_count:
               at_most:
                 "must not add up to more than %{count}. You have entered %{value} of these tags, so you must remove %{diff} of them."
+        kudo:
+          attributes:
+            commentable:
+              author_on_own_work: "You can't leave kudos on your own work."
+              blank: "We couldn't save your kudos, sorry!"
```

Comment:
Do we want to go with the `What did you want to leave kudos on?` error message for this case instead? It's not super helpful for the user, but it's more consistent with other error messages for when information is missing (a search for "what did you want" turns up a whole pile of messages).

Reply:
Works for me, I'll change it. Previously, the model will return "What did you want to leave kudos on?", but the controller and the front-end js will override it with "We couldn't save your kudos, sorry!".